src/main/java/com/linkedin/kmf/services/configs/JettyServiceConfig.java

```diff
@@ -20,12 +20,20 @@
   public static final String PORT_CONFIG = "jetty.port";
   public static final String PORT_DOC = "Port to be used by the Jetty service.";
 
+  public static final String JOLOKIA_URL_CONFIG = "jolokia.url";
+  public static final String JOLOKIA_URL_DOC = "URL to the Jolokia endpoint";
+
   static {
     CONFIG = new ConfigDef().define(PORT_CONFIG,
                                     ConfigDef.Type.INT,
                                     8000,
                                     ConfigDef.Importance.LOW,
-                                    PORT_DOC);
+                                    PORT_DOC)
+                            .define(JOLOKIA_URL_CONFIG,
```

Comment:
It seems a bit weird to have the number `8778` defined in both JettyServiceConfig and the JolokiaServiceConfig. Can we only define and use this in JolokiaServiceConfig?

Reply:
Please see my comment above for the dependency this would create between the two services.

Reply:
I'm not sure leaving the end user to carry this configuration detail in query params ( a bookmark or something similar ) would be the best. At minimum, it would be initially confusing.

I prefer "just works" :-)

Reply:
@andlaz I also prefer things to just work. But user already needs to explicitly specify the jetty port (e.g. 8000) anyway. So I am not sure how much different it makes to the user to not having to specify the jolokia port. We can just provide user a link (e.g. localhost:8000/index.html?jolokiaPort=8779&joloiaIP=localhost) that they can use to access the web GUI rather than giving them a port number and ask them figure out the link. And under this assumption, then it does not really matter whether there is one more parameter in the link. What do you think?

I am trying to see whether the additional code complexity is worth the convenience it gives to the user.